### PR TITLE
test pull_request_review_comment

### PR DESCRIPTION
**Pull request by daint-1849**

### CODE REVIEW

**Comment by daint-1849** [comment]
abc

**Review by daint-1849 — COMMENTED** [review]
abc

**Review by daint2git — DISMISSED** [review]

**Review by daint2git — COMMENTED** [review]
abc

File: .github/workflows/pull-request-review-comment.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: pull-request-review-comment
+on:
+  pull_request_review_comment:
+    branches:
+      - '**'
+    # types: [opened, labeled, unlabeled]
+jobs:
```
Comment (daint2git) [inline .github/workflows/pull-request-review-comment.yml:7]: 123435

**Review by daint2git — COMMENTED** [review]
ok

File: .github/workflows/pull-request-review-comment.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  say-hello:
```
Comment (daint-1849) [inline .github/workflows/pull-request-review-comment.yml:8]: mmm

**Comment by daint-1849** [comment]
abc

**Comment by daint2git** [comment]
abc

**Review by daint2git — APPROVED** [review]

**Pull request merged**